src/library/Uno.Material/Styles/Controls/v2/Button.xaml

```diff
@@ -13,21 +13,530 @@
 								 NotNullValue="0,0,2,0"
 								 NullValue="0" />
 
-	<!-- Brushes -->
-	<StaticResource x:Key="MaterialOutlinedButtonBorderBrush"
-					ResourceKey="OutlineBrush" />
-
 	<!-- Variables -->
-	<x:Double x:Key="MaterialButtonMinHeight">40</x:Double>
-	<x:Double x:Key="MaterialButtonMinWidth">40</x:Double>
-	<Thickness x:Key="MaterialButtonPadding">16,0</Thickness>
-	<Thickness x:Key="MaterialTextButtonPadding">12,0</Thickness>
-	<Thickness x:Key="MaterialButtonBorderThickness">0</Thickness>
-	<Thickness x:Key="MaterialOutlinedButtonBorderThickness">1</Thickness>
-	<CornerRadius x:Key="MaterialButtonCornerRadius">20</CornerRadius>
-	<x:String x:Key="MaterialButtonVerticalContentAlignment">Center</x:String>
-	<x:String x:Key="MaterialButtonHorizontalContentAlignment">Center</x:String>
-
+	<x:Double x:Key="ButtonMinHeight">40</x:Double>
+	<x:Double x:Key="ButtonMinWidth">40</x:Double>
+	<x:Double x:Key="ButtonIconMinWidth">18</x:Double>
+	<x:Double x:Key="ElevatedButtonElevation">1</x:Double>
+	<x:Double x:Key="ElevatedButtonElevationDisabled">0</x:Double>
+	<x:Double x:Key="ButtonElevation">0</x:Double>
+	<x:Double x:Key="IconButtonOpacityVisibleState">1</x:Double>
+	<x:Double x:Key="IconButtonOpacityHiddenState">0</x:Double>
+	<Thickness x:Key="ButtonPadding">16,0</Thickness>
+	<Thickness x:Key="TextButtonPadding">12,0</Thickness>
+	<Thickness x:Key="TextButtonIconMargin">0,0,8,0</Thickness>
+	<Thickness x:Key="ButtonBorderThickness">0</Thickness>
+	<Thickness x:Key="OutlinedButtonBorderThickness">1</Thickness>
+	<Thickness x:Key="ElevatedButtonMargin">0,0,0,1</Thickness>
+	<Thickness x:Key="ElevatedButtonDisabledMargin">0</Thickness>
+	<Thickness x:Key="ButtonContentMargin">8,0</Thickness>
+	<Thickness x:Key="ButtonMargin">0</Thickness>
+	<CornerRadius x:Key="ButtonCornerRadius">20</CornerRadius>
+	<x:String x:Key="ButtonVerticalContentAlignment">Center</x:String>
+	<x:String x:Key="ButtonHorizontalContentAlignment">Center</x:String>
+	<x:String x:Key="ButtonIconHorizontalAlignment">Stretch</x:String>
+	<x:String x:Key="ButtonIconVerticalAlignment">Stretch</x:String>
+	<x:String x:Key="IconButtonEllipseVerticalAlignment">Stretch</x:String>
+	<x:String x:Key="IconButtonEllipseHorizontalAlignment">Stretch</x:String>
+	<x:Boolean x:Key="ElevatedButtonIsTintEnabled">True</x:Boolean>
+
+	<ResourceDictionary.ThemeDictionaries>
+		<ResourceDictionary x:Key="Default">
+			<!--MaterialElevatedButtonStyle-->
```

Comment:
Looks good! Only thing is the formatting, if you run Xaml Styler it should fix this, unless it's overridden in your XamlStyler settings but the x:Key and the ResourceKey should be on the same line for the StaticResource.

And one last thing, I know its annoying but we should also group the StaticResources for each style by property. So 

```xml
<!--MaterialElevatedButtonStyle-->

ElevatedButtonForeground
ElevatedButtonForegroundPointerOver
ElevatedButtonForegroundDisabled
ElevatedButtonForegroundPressed

ElevatedButtonIconForeground
ElevatedButtonIconForegroundPointerOver
ElevatedButtonIconForegroundDisabled
ElevatedButtonIconForegroundPressed

etc...
```

Similar to what we do for other styles like [here](https://github.com/unoplatform/Uno.Themes/blob/d5d803370cc99e11e3e895992a220b95abec477a/src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml#L27-L50)